### ISSUE TITLE
Mismatch counts for associated WI with iterations 

### ISSUE THREAD

**Issue by naina-verma** [fabric8io/fabric8-planner#1567]
Mismatch counts for associated WI with iterations 

Steps to reproduce 
- Associate few WI with iterations 
- now check the associated WI count should update  in iterations  columns

Video Link: https://drive.google.com/a/redhat.com/file/d/0B1qUq_laG6COdnhsNTQ5NXc0V3M/view?usp=sharing

**Comment by naina-verma** [comment]
@vikram-raj @sanbornsen can you please look into it !

**Comment by vikram-raj** [comment]
@michaelkleinhenz Is your PR #1561 will fix this issue?

**Comment by pranavgore09** [comment]
This is being tracked [Core#1151](https://github.com/almighty/almighty-core/pull/1151)

**Comment by michaelkleinhenz** [comment]
@vikram-raj yes, it will when the core returns the correct numbers :-) So we need to wait for @pranavgore09's PR.
